contracts/Chorus.sol

```diff
@@ -236,6 +236,10 @@ contract Chorus is Inflation, OracleGetter, ERC20 {
         onlyAdmin
         within100e18Range(_amount) //between 0% and 10,000%
     {
+        require(
+            _amount > collateralRatio(),
```

Comment:
make just 100%...you want to be able to lower it...just not have a blanket zero

Reply:
Sounds good, I have updated the function so that the owner can set the collateral threshold no lower than 100% collateralization